### PR TITLE
Fix keyvault starter readme

### PR DESCRIPTION
**Pull request by moarychan**
Fixes Azure/azure-sdk-for-java#23587

Update a couple of grammatical errors.

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md [hunk 1]
```diff
@@ -25,19 +25,20 @@ From a developer's perspective, Key Vault APIs accept and return secret values a
 
 For highly sensitive data, clients should consider additional layers of protection for data. Encrypting data using a separate protection key prior to storage in Key Vault is one example.
 
-Key Vault also supports a contentType field for secrets. Clients may specify the content type of a secret to assist in interpreting the secret data when it's retrieved. The maximum length of this field is 255 characters. There are no pre-defined values. The suggested usage is as a hint for interpreting the secret data.
+Key Vault also supports a contentType field for secrets. Clients may specify the content type of secret to assist in interpreting the secret data when it's retrieved. The maximum length of this field is 255 characters. There are no pre-defined values. The suggested usage is as a hint for interpreting the secret data.
+
+Besides, this starter provides the features of supporting multiple Key Vaults, case-sensitive mode of Key Vault names, and using placeholder presenting Key Vault names in the property file.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md:30]: `Besides, this starter supports multiple Key Vaults`

Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md:30]: Could you please update this doc by the intellij's hints?
image

File: sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md [hunk 2]
```diff
@@ -48,17 +49,17 @@ in the Spring Boot documentation.
...
+- `azure.keyvault.secret-keys` is used to indicate that if an application using specific secret keys and this property is set, the application will only load the keys in the property and won't load all the keys from Key Vault, that means if you want to update your secrets, you need to restart the application rather than only add secrets in the Key Vault.
 - `azure.keyvault.authority-host` is the URL at which your identity provider can be reached.
    - If working with azure global, just left the property blank, and the value will be filled with the default value.
-   - If working with azure stack, set the property with authority URL.
+   - If working with azure stack, set the property with authority URI.
 - `azure.keyvault.secret-service-version`
-   - The valid secret-service-version value can be found [here][version_link]. 
-   - This property is optional, if property not set, the property will be filled with the latest value.
+   - The valid values can be found [here][version_link]. 
+   - This property is optional. If the property is not set, the property will be filled with the latest value.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md:62]: There is an item in the issue description: 

> Support for multiple key vaults is a great feature to have! How many are supported? Is it undefined?

Could you please fix this?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md (hunk 1 above)
Comment (moarychan) [reply sdk/spring/azure-spring-boot-starter-keyvault-secrets/README.md]: OK

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**